.github/workflows/test.yml

```diff
@@ -0,0 +1,14 @@
+name: test
```

Comment:
`test / tests (pull_request) ` it shows up like this in the list of tests. Could we give it better name? (Not that I know of any :) )

Reply:
Indeed - as per IRC, I renamed it to "bots", to coincide with `./test-bots` (the script that runs).